src/main/java/frc/robot/commands/arm/Retract.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands.arm;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.constants.ArmConstants;
+import frc.robot.subsystems.FourBarArm;
+
+public class Retract extends CommandBase {
+  FourBarArm m_arm;
+  PIDController m_pid;
+  public Retract(FourBarArm arm, PIDController pid) {
```

Comment:
Ca this class be implemented in terms of Extend Command?

Reply:
Yes. Retract = new ExtendToPosition(arm, 0). It could be made as a sequential command. Alternatively, do not use a Retract command.